MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -432,5 +454,5 @@ curl http://${host_ip}:${MEGA_SERVICE_PORT}/v1/multimodalqna  \
 ```bash
```

Comment:
Could you add a line above this noting that this is requesting and audio and text response using `"modalities": ["text", "audio"]`? It's easy to miss because the line is so long that in the README you have to scroll way to the side to even see it.